### DISCUSSION TITLE
In IntelliJ terminal how do I set the heights correctly so it lines up correctly

### DISCUSSION THREAD

**Discussion by trajano** [JanDeDobbeleer/oh-my-posh#4201] [Troubleshoot]
It seems with Caskaydia Cove NFM it's really bad

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/110627/91a9a926-7dc5-4ea1-9104-e307cfacf438)

With Meslo LGM it's not as bad but I can still see 1px overshoot and there are also the black bands

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/110627/e4a20636-8732-4452-8156-6366a07ecbe6)

I've tried different settings on this screen including line height `1.0`, `2.0`

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/110627/1a73368e-7eff-4b36-848c-ad1d02f88236)

But this person's screen looks ok https://github.com/JanDeDobbeleer/oh-my-posh/discussions/630#discussioncomment-595694

**Comment by JanDeDobbeleer** [comment]
I would ask @Ayfri directly :-)

**Reply by Ayfri** [reply]
Hi sorry, I don't have any idea how to fix this properly, I'm using JetBrainsMono NF font which is pretty good and working well, but it's not perfect and don't have the exact same render as in my terminal.
There is this [issue on YouTrack](https://youtrack.jetbrains.com/issue/IDEA-118832/unicode-character-is-incorrectly-displayed-in-terminal) speaking about that, but it seems to not be fixed for powerline characters.
